test/build-rpms

```diff
@@ -27,30 +27,41 @@ def build_rpms(srpm, image, verbose, quick):
     try:
         machine.start()
         machine.wait_boot()
-        machine.execute("su builder -c 'mkdir -p /var/tmp/build/SRPMS'")
-        vm_srpm = os.path.join("/var/tmp/build/SRPMS", os.path.basename(srpm))
-        machine.upload([os.path.realpath(srpm)], vm_srpm)
-
-        # build rpms
-        mock_opts = ("--verbose" if verbose else "") + (" --nocheck" if quick else "")
-        machine.execute("su builder -c 'mock --no-clean --disablerepo=* --offline --resultdir /var/tmp/build "
-                        f"{mock_opts} --rebuild /var/tmp/build/SRPMS/*.src.rpm'", timeout=1800)
 
         scenario = os.environ.get("TEST_SCENARIO")
-        if scenario:
-            if scenario.startswith("cockpit-pr-"):
-                cockpit_pr = scenario.split("-")[-1]
-                machine.execute(f"dnf copr enable -y packit/cockpit-project-cockpit-{cockpit_pr}", stdout=sys.stdout)
-            if scenario.startswith("anaconda-pr-"):
-                anaconda_pr = scenario.split("-")[-1]
-                machine.execute(f"dnf copr enable -y packit/rhinstaller-anaconda-{anaconda_pr}", stdout=sys.stdout)
-            else:
-                machine.execute(f"dnf copr enable @rhinstaller/Anaconda ", stdout=sys.stdout)
-
-        # download cockpit rpms
-        # FIXME boot.iso on rawhide does not currently contain the new cockpit dependencies
+        # Pull cockpit dependencies from the image default compose
+        # unless we are testing a PR on cockpit-project/cockpit, then pull it from the PR COPR repo
+        if scenario and scenario.startswith("cockpit-pr-"):
+            cockpit_pr = scenario.split("-")[-1]
+            machine.execute(f"dnf copr enable -y packit/cockpit-project-cockpit-{cockpit_pr}", stdout=sys.stdout)
+
+        # Build anaconda-webui from SRPM unless we are testing a PR, then pull it from the PR COPT repo
```

Comment:
I suspect you mean the opposite, "**if** we are testing a PR, otherwise pull it ..."? (this sentence implies it's an anaconda-webui PR) Because then you do want to build the rpms from the srpm that you just built. 

TBH the whole structure is a bit muddy: It seems the Makefile always builds an srpm from the checkout (PR for an anaconda-webui PR, main for a cockpit or anaconda PR), but the srpm gets ignored in the latter two cases, right? There could be a comment to point that out, that it is intended. Or better yet, shift all of this one level up, as for testing a cockpit or anaconda PR there shouldn't be any rpm build involved in the first place, so invoking a "build-rpms" script sounds weird.

Or perhaps rename "build-rpms" to "vm.install" or something, and move the whole setup here?